mlflow/recipes/utils/metrics.py

```diff
@@ -86,6 +100,26 @@ def logloss(true_label, predicted_proba, eps=1e-15):
     )
 
 
+def _get_extended_task(step_config: Dict[str, Any]) -> str:
+    """
+    :param step_config: Step config
+    :return: Extended type string. Currently supported types are: "regression",
+    "binary_classification", "multiclass_classification"
+    """
+    tmpl = step_config.get("recipe")
+    if "regression" in tmpl:
+        return "regression"
+    elif "classification" in tmpl:
+        if "positive_class" in step_config:
+            return "binary_classification"
+        else:
+            return "multiclass_classification"
```

Comment:
Can we call these `classification/binary` and `classification/multiclass` instead? Ultimately these should be very similar to the `recipe` field. I can imagine we might offer sub-recipes like `classification/binary/v1` and `classification/multiclass/v2`

Reply:
Great idea! Done.